doc/man3/CMS_verify.pod

```diff
@@ -24,8 +24,8 @@ CMS_get0_signers - verify a CMS SignedData structure
 CMS_verify() is very similar to L<PKCS7_verify(3)>. It verifies a
 B<CMS SignedData> structure contained in a structure of type B<CMS_ContentInfo>.
 I<cms> points to the B<CMS_ContentInfo> structure to verify.
-The optional I<certs> parameter refers to a set of certificates
-in which to search for signing certificates.
+The optional I<certs> parameter can provide a list of certificates,
+which is searched first for the signer's certificate.
```

Comment:
```suggestion
which is searched first for the signers certificates.
```

Reply:
changed to `which is searched first for signer certificates.`